minecraft_servers/justchunks/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "server_name": "justchunks",
+  "nice_name": "JustChunks",
+  "direct_ip": "mc.justchunks.net",
+  "server_wildcards": [
+    "%.justchunks.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en",
+    "es"
+  ],
+  "social": {
+    "web": "https://justchunks.net",
+    "web_shop": "https://shop.justchunks.net",
+    "web_support": "https://forum.timolia.de",
```

Comment:
This is not correct I guess? 🤔

Reply:
Thanks, this one slipped through :sweat_smile: